invenio_app_ils/ui/src/invenio_app_ils/pages/frontsite/PatronProfile/PatronCurrentDocumentRequests/state/actions.js

```diff
@@ -0,0 +1,52 @@
+import { IS_LOADING, SUCCESS, HAS_ERROR } from './types';
+import { documentRequest as documentRequestApi } from '../../../../../common/api';
+import { sendErrorNotification } from '../../../../../common/components/Notifications';
+import { ES_DELAY } from '../../../../../common/config/';
+
+const selectQuery = (patronPid, page = 1) => {
+  return documentRequestApi
+    .query()
+    .withPatronPid(patronPid)
+    .withPage(page)
+    .withState('PENDING')
```

Comment:
there is a config to avoid this `PENDING`, `circulation.loanRequestStates`

Reply:
but that is for Loans, and I need for DocumentRequest :) I will create config for this